operator/src/main/java/oracle/kubernetes/operator/PodWatcher.java

```diff
@@ -35,7 +40,7 @@
  */
 public class PodWatcher extends Watcher<V1Pod> implements WatchListener<V1Pod>, PodAwaiterStepFactory {
   private static final LoggingFacade LOGGER = LoggingFactory.getLogger("Operator", "Operator");
-
+  private static final String DOMAIN_INTROSPECTOR_JOB_SUFFIX = "introspect-domain-job";
```

Comment:
There is a DOMAIN_INTROSPECTOR_JOB_PATTERN in LegalNames.java where the job names come from. It is probably not a good idea to have 2 constants for introspector job name in 2 different classes.

Reply:
fixed